diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+For a website to connect to the web which allows users to interact throught a site.
```

Comment:
`One purpose of a backend is to allow users on the same web app to interact with each other.
Without a backend, users would not be able to do this, limiting what our application is capable of. Another
purpose is to securely store data that can persist over time.`